app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -1,5 +1,58 @@
 import * as yup from "yup";
 
+export const NOTES_LIST_DATA = [
+  {
+    id: 1,
+    title: "How to claim the warranty?",
+    description: `Are you getting my texts???" she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting`,
+    tag: "Getting Started",
+    status: "created",
+    createdAt: "2 hours ago",
+    createdBy: "Oliver Smith",
+    imageUrl: "https://i.pravatar.cc/300",
+  },
+  {
+    id: 2,
+    title: "How to claim the warranty?",
+    description: `Are you getting my texts???" she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting`,
+    tag: "Onboarding",
+    status: "drafted",
+    createdAt: "2 hours ago",
+    createdBy: "Oliver Smith",
+    imageUrl: "https://i.pravatar.cc/300",
+  },
+  {
+    id: 3,
+    title: "How to claim the warranty?",
+    description: `Are you getting my texts???" she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting`,
```

Comment:
Why is this a template string even when you aren't performing any interpolation?